src/Cli/Microsoft.TemplateEngine.Cli/TemplatePackageCoordinator.cs

```diff
@@ -208,7 +208,8 @@ internal async Task<NewCommandStatus> EnterInstallFlowAsync(InstallCommandArgs a
 
             foreach (string installArg in args.TemplatePackages)
             {
-                string[] split = installArg.Split(["::"], StringSplitOptions.RemoveEmptyEntries).SelectMany(arg => arg.Split('@', StringSplitOptions.RemoveEmptyEntries)).ToArray();
+                bool isPath = File.Exists(installArg) || Directory.Exists(installArg);
```

Comment:
Isn't there something to check if it is a path? Like, something related to that RelativePath utility thing. Because you shouldn't need to check if something is on-disk to know if the argument is a path or not.

Reply:
We could call `http://msdn.microsoft.com/en-us/library/system.io.path.getfullpath.aspx` Path.GetFullPath and catch the exception to see if it's not a path?

Reply:
I don't know that I agree with this. If you want to dotnet new install something in your current directory, it would be very hard to tell (without checking if it exists) that it's a path.

Reply:
And if you were to just check for an exception from Path.GetFullPath, I think ~any package would show up as a path